templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/includes/new_public_participation_statement_confirm_content.html.twig

```diff
@@ -37,7 +37,9 @@
     {% block user_data_hint %}
         <p>
             {% if projectType == 'gateway' %}
-                {{ "explanation.data.user.change"|trans({ href: gatewayURL, label: 'gateway'|trans })|wysiwyg }}
+                {% if not hasPermission('feature_show_notice_draft_statement') %}
+                    {{ "explanation.data.user.change"|trans({ href: gatewayURL, label: 'gateway'|trans })|wysiwyg }}
+                {% endif %}
```

Comment:
sorry, one more.. you can also reduce the number of if statements by adding the permission check in the first if statement --> `{% if projectType == 'gateway' and not hasPermission('feature_show_notice_draft_statement') %}`